int32 feat_id
geometry_msgs/Point[] pts  # point locations
geometry_msgs/Vector3[] displacements # displacement of each feature from the previous frame
#geometry_msgs/Point[] norm_pts  # normalized point locations
#int32[] img_ids  # seq of image per point location